src/base/_bootstrap-4.scss

```diff
@@ -0,0 +1,34 @@
+/*
+  WET-BOEW
+  @title: Bootstrap 4 suplemental style for WET-BOEW
+ */
+
+/* Padding */
+.p-0 {
+	padding: 0 !important;
+}
+
+.pl-2
+.px-2 {
+	padding-left: 5px !important;
```

Comment:
Doesn't the `.px-2` also have a `padding-right: 5px !important;`?

Reply:
It originally had `padding-left: 0.5rem !important;` so should be 8px.

Reply:
I put the same as the value for ```mrgn-bttm-sm```